private/buf/cmd/buf/command/convert/testdata/convert/bin_json/duration.txtpb

```diff
@@ -1 +1 @@
-seconds:  3600
+seconds: 3600
```

Comment:
for some reason there are diff for txtpb format re: spaces, but shouldn't has any actual affect